### DISCUSSION TITLE
Where can I download the VSIX for the extension

### DISCUSSION THREAD

**Discussion by The-Experimentalist** [OmniSharp/omnisharp-vscode#6860] [Q&A] [answered]
Hi

I can't find the vsix file for this extension anywhere, I was expecting to find it on the releases page. I am trying to install the extension on a network isolated server so need the vsix file.

Can anyone help?

**Comment by The-Experimentalist** [comment]
![OmniSharp for Windows ( NET 4 6  x64) VS Code Error](https://github.com/dotnet/vscode-csharp/assets/9028834/e67f9e44-f665-4941-820d-bbd3b3a23faf)

**Comment by gregg-miskelly** [comment, marked as answer]
See the 'Download extension' link in the [Visual Studio Code extension gallery](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)

**Reply by The-Experimentalist** [reply]
Hi Gregg

Thanks for that. I am somewhat confused. I had already downloaded and installed that but I don't then understand why I get the error in the above screenshot. The URL in the screenshot doesn't even seem to exist. It says "Downloading package "Omnisharp for Windows (.Net 4.6 / x64) is that part of the "C# for Visual Studio Code" extension. It is all a puzzle to me. Do you have any ideas.

Regards
Ex.

**Reply by gregg-miskelly** [reply]
I *think* what is going on is you are using a very old version of the extension, as the extension hasn't had a "OmniSharp for Windows (.NET 4.6 / x64)" dependency since #4961 back in 2021.
